configs/starfive/visionfive2/buildroot/uboot

```diff
@@ -1,13 +1,6 @@
 BR2_TARGET_UBOOT=y
 BR2_TARGET_UBOOT_BUILD_SYSTEM_KCONFIG=y
-
-# BR2_TARGET_UBOOT_LATEST_VERSION=y
-BR2_TARGET_UBOOT_CUSTOM_TARBALL=y
-
-# branch: JH7110_VisionFive2_devel
-# source: https://github.com/starfive-tech/u-boot/tree/JH7110_VisionFive2_devel
-# 59cf9af678dbfa3d73f6cb86ed1ae7219da9f5c9
-BR2_TARGET_UBOOT_CUSTOM_TARBALL_LOCATION="$(call github,starfive-tech,u-boot,ac0ac696256abf412826d74ee918dd417e207d7b)/u-boot-visionfive2-ac0ac6-r2.tar.gz"
+BR2_TARGET_UBOOT_LATEST_VERSION=y
```

Comment:
Could we set a specific version instead of LATEST? I usually do that to avoid random breakages when updating buildroot. This way someone can update it when needed and test to make sure the new version works properly before committing.

Reply:
@maxberger I think you still have latest version set here as well as the custom version. Probably need to delete this line - LATEST_VERSION